projects/project_ideas.md

```diff
@@ -0,0 +1,59 @@
+# CS 7290: Project Ideas
+**Prof. Robert Ness**
+
+## 1. State-of-the-art Causal Inference methods
```

Comment:
Put this at the bottom of the page and make it a "With permission from the instructor" thing.